ui/lib/apps/TopSQL/pages/List/ListChart.tsx

```diff
@@ -79,6 +79,7 @@ export const ListChart = forwardRef<Chart, ListChartProps>(
         />
         <Axis
           id="left"
+          title="CPU Time"
```

Comment:
Seems that this is not calling i18n functions. Will it cause problems?

Reply:
That's right! Using i18n will be better.